__mocks__/react-relay.js

```diff
@@ -2,13 +2,20 @@
 // Should be used in tests as a replacement for `react-storybooks-relay-container`
 //
 var React = require.requireActual("react")
-var Relay = require.requireActual("react-relay")
+
+const graphql = (strings, ...keys) => {
```

Comment:
Mind adding a small comment about what this does?

Reply:
Done :+1:  https://github.com/artsy/reaction/pull/3002/commits/6e0b7f2873fb8ede6bfd8dbf29b30200330b2662